github/lib/configure_repo.rb

```diff
@@ -65,17 +65,37 @@ def update_webhooks
 
   def required_status_checks
     if jenkinsfile_exists?
-      {
-        strict: false, # "Require branches to be up to date before merging"
-        contexts: ["continuous-integration/jenkins/branch"]
-      }
+    {
+      strict: false, # "Require branches to be up to date before merging"
+      contexts: [
+        jenkinsfile_exists? ? "continuous-integration/jenkins/branch" : nil,
```

Comment:
This check isn't required since `jenkinsfile_exists?` is checked in the `if` statement